modules/mono/glue/cs_files/Transform.cs

```diff
@@ -104,6 +104,12 @@ public Transform(Vector3 xAxis, Vector3 yAxis, Vector3 zAxis, Vector3 origin)
             this.origin = origin;
         }
 
+        public Transform(Quat quat, Vector3 origin)
+        {
+            this.basis = new Basis(quat);
+            this.origin = origin;
+        }
+
```

Comment:
I don't see this anywhere in the Godot API either.

Reply:
That also would be need to be added then. It's there in GDScript

Reply:
I don't see think constructor in GDScript.

Reply:
But there is

Reply:
If there are other missing constructors from Basis in Transform those should also be added in core/math so don't worry I'll add any missing ones

Reply:
Of course they'll have origin as the extra parameter

Reply:
So in GDScript, Transform(quat, origin) does work but when I checked the documentation, the closest constructor listed there is Transform(Basis, Vector3). I don't know if this is a documentation mistake or not, but if it's not, it seems GDScript can automatically convert a Basis to Quat?

That constructor should be added anyway though, I'm just surprised GDScript could do that